app/recommender_engine_stub.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+
+
+class RecommenderEngineStub:
+    @classmethod
+    def get_recommendations(cls, context, location, version):
+        if version not in ["v1", "v2"]:
+            raise InvalidRecommendationPanelVersionError
+
+        if location not in ["services_list"]:
```

Comment:
Generally i this kind of situations immutable types (tuples) are recommended `("v1", "v2)`, etc.

Reply:
fixed